### PR TITLE
427 habitatslideshowtaggedpics

### PR DESCRIPTION
**Pull request by kathleenahall**

### CODE REVIEW

File: src/containers/HabitatProject/HabitatProjectContent.tsx [hunk 1]
```diff
@@ -1,23 +1,25 @@
...
-},
-{
-  _id: '5e7273f7349ad8000410f762',
-  title: 'Psalm 127',
-  url: 'https://dl.dropboxusercontent.com/s/w4drwppxa9jrcmo/psalm127.jpg?dl=0',
-  type: 'habitat',
-  comments: '',
-},
-]; // 'showCaption' displays captions if needed'
+// const images = [{
```
Comment (JoshuaVSherman) [inline src/containers/HabitatProject/HabitatProjectContent.tsx:7]: @kathleenahall please delete the code that you commented out

**Review by JoshuaVSherman — CHANGES_REQUESTED** [review]

File: src/containers/HabitatProject/HabitatProjectContent.tsx (hunk 1 above)
Comment (kathleenahall) [reply src/containers/HabitatProject/HabitatProjectContent.tsx:7]: How do I know what the codeclimate issue is?

Comment (JoshuaVSherman) [reply src/containers/HabitatProject/HabitatProjectContent.tsx:7]: @kathleenahall scroll down and click the details, it has to do with unit test coverage

Comment (JoshuaVSherman) [reply src/containers/HabitatProject/HabitatProjectContent.tsx:7]: @kathleenahall it looks like details does not show you the test coverage, you need to run the tests locally then open up the /coverage/lcov-report/index.html file in our chrome browser

File: src/containers/HabitatProject/HabitatProjectContent.tsx [hunk 2]
```diff
@@ -197,45 +202,63 @@ const FamilyInfo = () => (
...
-        </TeamLeaders>
-      </HabitatGridItem>
+function getImagesFromStorage(setHabitatImages:(arg0: Ibook[] | null) => void) {
+  try {
+    const data = sessionStorage.getItem('persist:root');
+    const json = JSON.parse(data || '');
+    const { otherPics } = json;
+    const pics = JSON.parse(otherPics);
+    const images = pics.otherPics.filter((d:Ibook) => d.type === 'habitat');
+    console.log(images);
```
Comment (JoshuaVSherman) [inline src/containers/HabitatProject/HabitatProjectContent.tsx:212]: delete console.log here, line 212

**Review by JoshuaVSherman — CHANGES_REQUESTED** [review]

File: src/containers/HabitatProject/HabitatProjectContent.tsx (hunk 1 above)
Comment (kathleenahall) [reply src/containers/HabitatProject/HabitatProjectContent.tsx:7]: npm version minor is throwing me an error so I'm still working on that, this is what the coverage shows but I'm unsure of what to do next

Screenshot from 2022-10-31 09-53-33

Comment (kathleenahall) [reply src/containers/HabitatProject/HabitatProjectContent.tsx:7]: Nvm, I got the version updated

Comment (JoshuaVSherman) [reply src/containers/HabitatProject/HabitatProjectContent.tsx]: @kathleenahall i'm going to approve this merge request, but you will need to next work on learning how to write the unit tests, the above is a unit test coverage report, you can click into the report and it shows you exactly which functions are not having any unit tests. Those will need to be written. Start reading about `react unit testing with jest`

**Pull request merged**